### ISSUE TITLE
Crazy ride

### ISSUE THREAD

**Issue by unionjac13** [teknogods/OpenParrot#276]
Did extensive testing on this I have several systems. Crashed on I7 intel 11 gen 's with 3060tigpu while in game after 20 seconds or so 7 times
worked on I5 intel with ibm gpu built in but speed was too slow hard to steer and gas (speed) barely moved 
never crashed 
Amd Processor Nvidia 3060ti gpu never crashed

**Comment by Pete-McDonald** [comment]
Yeah I have the same stability issue, it's crashes pretty consistently - each time progressing a little further.   I can now JUST complete the first fare - then it crashes.  I haven't been able to get past there.

1. I've tried setting the .exe to run as Admin and Compatibility mode (Win 8) with no change
2. Tried windowed, full screen and custom resolution in TP with no change
3. I've tried installing it at D:\SysStart - no change.

SPECS:

- i9 12900K
- Nvidia 3080Ti
- TP is fully up-to-date

Happy to share anything that might help with diagnosing this

**Comment by EmiMidnight** [comment]
Can you try again with the newest TPUI update that just got uploaded?

**Comment by Pete-McDonald** [comment]
> Can you try again with the newest TPUI update that just got uploaded?

Hey Emi, thanks for the quick reply. Updated TP to 1.0.0.1225:

Game crashed earlier than usual, just after you get the first notification to use Nitro Boost.
Restarted, game ran to the end of Fare 1 (usual place) then crashed.

One thing I noticed, that I swear wasn't happening before, was that my car was breaking as well as accelerating.  I'll do a re-calibration and try again - but that's probably not related.

Let me know if there's any logs or anything I can provide.

**Comment by Pete-McDonald** [comment]
Hey @EmiMidnight,

So small update, I've noticed that the game seems to crash ~45 seconds from when the UNIS logo appears.  It doesn't matter if I'm in the test menu, in game or just letting the attract mode run.  Almost exactly 45 seconds after the UNIS screen appears, it crashes regardless.

I tried upping the game time from 90 seconds to 600 but that didn't seem to change anything.

Thanks again for all your help on this.

**Comment by Pete-McDonald** [comment]
Confirm fixed in most recent update!  Thanks Emi, you're a straight up genius
